### ISSUE TITLE
skew props unclear

### ISSUE THREAD

**Issue by maximilianMairinger** [ismailman/decompose-dommatrix#1]
How do you interpret the values of `skewXY`? In your [transform-values-to-transform-commands](https://github.com/ismailman/transform-values-to-transform-commands) repo you simply replace the value with `skew`. While this is somewhat representable at smaller values (like `10deg`), the higher the value gets the extremer the inconsistency does.

Is this information simply omitted in matrices and cant be decomposed whatsoever? Or is this an aproximation for performance reasons? If so, is there a way to increase the precision (without completely breaking performance)?

**Comment by thednp** [comment]
I was scratching my head around the same issue, also the original MatrixMath script does that. 

I was experiencing with some method to use a WebkitCSSMatrix/DOMMatrix init at hand to update it once rotations and translations are extracted to further try and extract perspective and skews, which sometimes it works when a certain order of transform functions work, but never works perfect.

At this point, I'm still convinced there is no such thing as a reliable solution.

**Comment by patrickxia** [comment]
Yeah, the values of skewXY/skewYZ/skewXZ didn't seem useful at all until I realized they aren't in degrees and the conversion "from radians" doesn't make any sense.

The thing that I found useful was https://www.w3.org/TR/css-transforms-2/#recomposing-to-a-3d-matrix which tells you what to do with the skew. Essentially the reverse transformation is

```
matrix3d(1, 0, 0, 0, skewXY, 1, 0, 0, skewXZ, skewYZ, 1, 0, 0, 0, 0, 1)
```

but you have to use the real skew values as opposed to the ones that have gotten messed up by RAD_TO_DEG here: https://github.com/ismailman/decompose-dommatrix/blob/5fb3ee22cad37d14dbc4691b1b1eecff7f595802/decomposeMatrix.mjs#L129

I'll send a pull request shortly to get rid of the RAD_TO_DEG.
